lucetc/src/error.rs

```diff
@@ -32,7 +32,7 @@ pub enum Error {
     #[error("Artifact: {1}. {0:?}")]
     ArtifactError(ArtifactError, String),
     #[error("Failure: {1}. {0:?}")]
-    Failure(failure::Error, String),
+    Failure(anyhow::Error, String),
```

Comment:
not an issue with this PR, but we should take another pass at this enum with the latest changes that got rid of `failure`. This is the correct fix for now.